runtime/src/main/java/org/wildfly/swarm/config/runtime/invocation/Types.java

```diff
@@ -131,6 +131,11 @@ else if (ModelType.LIST == modelType) {
 
         return result;
     }
+    
+    public static String resolveListValueType(ModelNode value) {
```

Comment:
It seems the `resolveValueType` method isn't used anywhere else, so not sure if this wrapper is worth it.

Reply:
Agreed :-)

Reply:
Ah you already commented on this. TBH, since the underlying method can return a `?`, which only ever makes sense for generics, I don't think the wrapper makes a lot of sense.